README.md

```diff
@@ -2,6 +2,17 @@
 
 Welcome to Minicom! A prototype Intercom service. This is a project to help you set up for your interview later on. Inside this directory you'll find a foo website, a bar website, and servers that represent a simplified version of how Intercom works.
 
+# What do I have to do?
```

Comment:
@patocallaghan Do you think this is more explicit?

Reply:
Nice, I like this 👍